.github/workflows/ci.yml

```diff
@@ -94,15 +94,19 @@ jobs:
   windows:
     runs-on: windows-latest
 
-    name: "Windows (Swift 5.5)"
+    strategy:
+      matrix:
+        include:
+          - tag: 5.5-DEVELOPMENT-SNAPSHOT-2021-05-09-a
+            branch: swift-5.5-branch
```

Comment:
Great idea factoring the Swift snapshot into a variable.

Reply:
Lessons learned from Swift/Win32 :)